content/cif-connector/src/main/content/jcr_root/apps/commerce/gui/content/products/bindproducttreewizard/customfields/cqConf/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I had a quick check with Mark, actually you can completely remove that `customfields` part: this was made to add specific fields only for the Magento config (storeview and cat id), but we don't need that any more. You could just add that field directly in the wizard panel.

Reply:
+1